### PR TITLE
contrib/init/openrc: allow separate logs for stdout and stderr

### PR DESCRIPTION
**Pull request by williamh**
**- What I did**

Allow separate log files for stdout and stderr in the OpenRC scripts.

**- How I did it**

This was done by creating two new configuration variables, DOCKER_ERRFILE and DOCKER_OUTFILE in the conf.d file. If either of these is set when docker is started, they will be used to log stdout and stderr instead of using DOCKER_LOGFILE. If either or both of them is not set, DOCKER_LOGFILE will be used.

**- How to verify it**

If you change either of the mentioned settings before starting docker, you will find that the respective files are used for logging stdout and stderr instead of DOCKER_LOGFILE.

**- Description for the changelog**

Allow stdout and stderr to be logged separately if Docker is started with the OpenRC script.

### CODE REVIEW

**Comment by thaJeztah** [comment]
ping @tianon @andrewhsu PTAL

**Comment by aauren** [comment]
This change would be very beneficial to my workflow. Is there any chance that we can get this minor logging change reviewed today?

**Comment by thaJeztah** [comment]
@aauren if you need this today, you can simply update your local init script with the changes from this PR. Even if it's reviewed/merged soon, you'll have to wait for the next release for it to find its way into the released packages.

**Comment by williamh** [comment]
I am one of the packagers of Docker for Gentoo Linux and the primary maintainer of OpenRC. My plan is to do a revision bump once I know the changes here are merged into Docker so that we won't have to wait for the next release. I would rather not do the revision bump on Gentoo until I know for sure whether these changes will be accepted upstream.

Thanks,

William

**Comment by tianon** [comment]
If @williamh says this is correct, this is correct. :smile:  (as he mentioned, he's one of the OpenRC upstream maintainers)

Additionally, the default behavior remains unchanged. :+1:

LGTM

**Review by thaJeztah — APPROVED** [review]
works for me!

LGTM, thanks!

**Pull request merged**
